packit/api.py

```diff
@@ -216,7 +215,7 @@ def sync_release(
                     Path(self.up.local_project.working_dir),
                     Path(self.dg.local_project.working_dir),
                 )
-                sync_files(raw_sync_files)
+                sync_files(self.dg.local_project.working_dir, raw_sync_files)
 
```

Comment:
...in here actually, I picked a wrong method in my previous comment.

A little bit of context about my reasoning:

My concern is that the function `sync_files` is a well-defined and clear to
understand function. This PR adds a signifficant side-effect to the function -
creating a new file in the target destination - suddenly the function is harder
to read, harder to test etc. And since we use it also to copy from dist-git to
upstream, it would also create readme files during that process, which
something we don't want.

The important thing here is that the functionality of creating the readme file
is specific to upstream → dist-git, so I'd say that code should be placed in a
method which solves the problem.

We can discuss in person if that makes things more clear.